01/Anthlis/wordvalue.py

```diff
@@ -0,0 +1,49 @@
+import os
+import urllib.request
+
+# PREWORK
+DICTIONARY = os.path.join('/tmp', 'dictionary.txt')
+urllib.request.urlretrieve('http://bit.ly/2iQ3dlZ', DICTIONARY)
+scrabble_scores = [(1, "E A O I N R T L S U"), (2, "D G"), (3, "B C M P"),
+                   (4, "F H V W Y"), (5, "K"), (8, "J X"), (10, "Q Z")]
+LETTER_SCORES = {letter: score for score, letters in scrabble_scores
+                 for letter in letters.split()}
+
+
+def load_words():
+    """load the words dictionary (DICTIONARY constant) into a list and return it"""
+    word_list = list()
+    with open(DICTIONARY, 'r') as f:
+    # with open('dictionary.txt', 'r') as f:
+        for line in f:
+            word_list.append(line.strip())
+    return word_list
+
+
+def calc_word_value(word):
+    """given a word calculate its value using LETTER_SCORES"""
+    result = 0
+    # LETTER_SCORES is a dict so loop over and get the dict value each time +=
+    for letter in word:
+        result += LETTER_SCORES.get(letter.upper(), 0)
```

Comment:
use list comprehensions to further shorten your code. With list comprehensions you only need one line of code `result = [LETTER_SCORES.get(letter.upper(), 0) for letter in word]`